server/db/models/index.js

```diff
@@ -1,18 +1,10 @@
 const User = require('./user')
+const Mug = require('./mug')
 
-/**
- * If we had any associations to make, this would be a great place to put them!
- * ex. if we had another model called BlogPost, we might say:
- *
- *    BlogPost.belongsTo(User)
- */
+User.belongsToMany(Mug, {through: 'order'})
+Mug.belongsToMany(User, {through: 'order'})
```

Comment:
reformat to what barish wrote --> 
`User.belongsToMany(Mug, { as: "cart", through: "UserMug", foreignKey: "userId" });
Mug.belongsToMany(User, { as: "users", through: "UserMug", foreignKey: "mugId" });`